rpcs3/Emu/Cell/Modules/cellAudio.cpp

```diff
@@ -813,7 +814,7 @@ void cell_audio_thread::operator()()
 
 			if (time_left > cfg.period_comparison_margin)
 			{
-				thread_ctrl::wait_for(get_thread_wait_delay(time_left));
+				thread_ctrl::wait_until(&sleep_until, get_thread_wait_delay(time_left));
```

Comment:
I don't think this is correct. I think get_thread_wait_delay already calculates the exact time we need (I believe)

Reply:
It should probably be rewritten to use it, but I won't do it in this pr.